### PR TITLE
[C-3004] Scroll to top when switching tracks in upload

### PR DESCRIPTION
**Pull request by amendelsohn**
### Description

- Prev/next buttons scroll to top
- Clicking track or deleting track in MultiTrackSidebar scrolls to top

Using react provider to pass the scroll call down.

### How Has This Been Tested?
local web

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/amendel-upload-scroll-to-top

**Review by Kyle-Shanks — APPROVED** [review]
good stuff. might have missed it, but does this scroll to the top on failed submit also? thought that was part of the ticket

File: packages/web/src/pages/App.js [hunk 1]
```diff
@@ -698,7 +698,9 @@ class App extends Component {
                 <DesktopRoute
                   path={UPLOAD_PAGE}
                   isMobile={isMobileClient}
-                  component={UploadPage}
+                  render={(props) => (
+                    <UploadPage {...props} scrollToTop={this.scrollToTop} />
```
[the remark this answers is not in the document]
Comment (dylanjeffers) [reply packages/web/src/pages/App.js:702]: Love that idea

**Review by dylanjeffers — APPROVED** [review]
If you're down to put the scrollToTop into app context, that would be swell

File: packages/web/src/pages/App.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (amendelsohn) [reply packages/web/src/pages/App.js:702]: Gonna merge this and follow up with a separate PR Monday

**Comment by amendelsohn** [comment]
> good stuff. might have missed it, but does this scroll to the top on failed submit also? thought that was part of the ticket

Oh, don't think so. Good call. I'll add a follow-up item for that

**Pull request merged**